README.md

```diff
@@ -12,7 +12,7 @@ Name                                      | Description
 ------------------------------------------|----------
 <kbd>exunit-verify-all</kbd>              | Run all the tests in the current project
 <kbd>exunit-verify-all-in-umbrella</kbd>  | Run all the tests in the current umbrella project
-<kbd>exunit-verify</kbd>                  | Run all the tests in the current buffer
+<kbd>exunit-verify</kbd>                  | Run all the tests associated with the current buffer
```

Comment:
I think this description is difficult to understand without digging in further. Could it be extended to 
```suggestion
<kbd>exunit-verify</kbd>                  | Run all the tests in the current buffer, or the test file corresponding to the current buffer.
```

I think that will be easier to understand on first reading. And hopefully it's not too long

Reply:
Yes, that's definitely better. Thanks for the suggestion!